pages/a/zoomdata-xx.html

```diff
@@ -0,0 +1,64 @@
+title: Zoom Data XX
+link: <https://crate.io/a/zoomdata-xx>
+author: Chris Ward
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+created: 2016/06/15
+post_name: zoomdata-xx
+status: publish
+post_type: post
+tags: xx
+category: news, developernews
+
+ZoomData helps you query and visualize your big data sets to gain valuable insights quickly and easily. The (beta) Crate connector supports all Crate data types, including objects.
+
+## Add Data Source
+
+Click the hamburger menu item on the left hand side of the Zoom Data dashboard, then click *Data Sources* and *Edit Data Sources* and select the Crate data source.
```

Comment:
the *Data Sources* and *Edit Data Sources* parts of the sentence show up as _Data Sources_ for whatever reason ...